sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/CosmosTableTests.cs

```diff
@@ -60,6 +62,7 @@ public async Task TearDown()
 
         [Test]
         [RecordedTest]
+        [[Ignore("Flaky test: Need diagnose the table API issue from RP team")]
```

Comment:
```suggestion
        [Ignore("Flaky test: Need diagnose the table API issue from RP team")]
```
You have an extra `[`.